game_patch/object/entity.cpp

```diff
@@ -201,6 +202,79 @@ CodeInjection entity_process_pre_hide_riot_shield_injection{
     },
 };
 
+FunHook<void(int)> entity_blood_throw_gibs_hook{
+    0x0042E3C0, [](int handle) {
+        if (g_game_config.gibbing && !rf::is_multi) {
+            rf::Object* objp = rf::obj_from_handle(handle);
+
+            if (!objp) {
+                return; // invalid object
+            }
+
+            rf::Vector3 pos = objp->pos;
+            rf::GRoom* room = objp->room;
+            int explosion_vclip = 30;
+            int chunk_explosion_vclip = 30;
+            float explosion_vclip_radius = 1.0f;
+            rf::String debris_filename = "df_meatchunks0.V3D";
+
+            int debris_max_lifetime = 7000;
+            float debris_velocity = 8.5f;
+            float damage_scale = 1.0f;
+            rf::String cust_snd_set = "gib bounce";
```

Comment:
unnecessary allocation here, you can probably add `static const` here

Reply:
Unless I'm misunderstanding, I believe the current way is appropriate here.

`debris_spawn_from_object` expects `cust_snd_set` as `rf::String*`

Am I misunderstanding you?

Reply:
Oh, my bad. I thought you were only talking about the last line. 

I've added `static const` to the int and both floats.

Reply:
> I did add static though.

that was what I asked. with `static` it is allocated once and allocation is a relatively heavy operation

> I've added static const to the int and both floats.

I didn't ask for that but they can stay